src/pages/workspaces/List.js

```diff
@@ -329,14 +329,15 @@ export const WorkspaceList = _.flow(
           viewToggleButtons
         ]),
         div({ style: { display: 'flex', marginBottom: '1rem' } }, [
-          div({ style: { ...styles.filter, marginLeft: 'auto' } }, [
+          div({ style: { marginLeft: 'auto', display: 'flex', alignItems: 'center', fontSize: '1rem' } }, ['Filter by']),
+          div({ style: styles.filter }, [
             h(Select, {
               isClearable: true,
               isMulti: true,
               isSearchable: true,
               value: tagsFilter,
               hideSelectedOptions: true,
-              placeholder: 'Filter by tags',
+              placeholder: 'tags',
```

Comment:
I think we should capitalize these. Thoughts?

Reply:
Agreed, I think it looks better capitalized